plugins/completion/cmp/sources/cmp-ai.nix

```diff
@@ -0,0 +1,91 @@
+{
+  lib,
+  helpers,
+  config,
+  ...
+}:
+with lib;
+let
+  cfg = config.plugins.cmp-ai;
+in
+{
+  meta.maintainers = [ maintainers.GaetanLepage ];
+
+  options.plugins.cmp-ai = {
+    settings = helpers.mkSettingsOption {
+      description = "Options provided to the `require('cmp_ai.config'):setup` function.";
```

Comment:
Typo

```suggestion
      description = "Options provided to the `require('cmp_ai.config').setup` function.";
```

Reply:
Not this time !
https://github.com/tzachar/cmp-ai?tab=readme-ov-file#setup